openhands/runtime/client/client.py

```diff
@@ -711,7 +714,8 @@ async def upload_file(
         file: UploadFile, destination: str = '/', recursive: bool = False
     ):
         assert client is not None
-
+        print('CLIENT AVAILABLE' if client is not None else 'CLIENT UNAVAILABLE')
+        print('DESTINATION: ', destination)
```

Comment:
Can we replace this with logger.debug? Or remove it directly

Reply:
Woops! Leftovers